service/qos/service_level_controller.cc

```diff
@@ -231,15 +231,34 @@ future<std::optional<service_level_options>> service_level_controller::find_serv
 }
 
 future<>  service_level_controller::notify_service_level_added(sstring name, service_level sl_data) {
-    _service_levels_db.emplace(name, sl_data);
-    return make_ready_future();
+    return seastar::async( [this, name, sl_data] {
+        _subscribers.for_each([name, sl_data] (qos_configuration_change_subscriber* subscriber) {
+            try {
+                subscriber->on_before_service_level_add(name, sl_data.slo).get();
+            } catch(...) {
+                sl_logger.error("notify_service_level_added: exeption occured in one of the obesrvers callbacks");
```

Comment:
nitpicks: `s/exeption/exception`, `s/occured/occurred`, `s/obesrvers/observers`

Reply:
Also: log the exception

catch needs a space after it (it's not a function)